pkg/cloud/openstack/clients/machineservice_test.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package clients
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/gophercloud/utils/openstack/clientconfig"
+)
+
+func TestMachineServiceInstance(t *testing.T) {
+	clientOpts := new(clientconfig.ClientOpts)
+	cloud, err := clientconfig.GetCloudFromYAML(clientOpts)
+	if err != nil {
+		t.Errorf("Couldn't create clientconfig")
+	}
+	_, err = NewInstanceService(cloud)
+	if !(strings.Contains(err.Error(), "i/o timeout")) {
```

Comment:
@Lion-Wei @chaosaffe I was helping @iamemilio with this test and this looks like a good start, although it's probably not the best way to do it. We should use `mock` instead, or just write the test differently.